docs/installation.rst

```diff
@@ -24,22 +23,14 @@ after which you can simply install without specifying the channels::
     `Anaconda scientific python distribution <https://www.anaconda.com/products/individual>`_
     to manage your numerical and scientific Python packages.
 
-Install with `pip <https://pypi.org/project/pip/>`_
----------------------------------------------------
-::
-
-    $ pip install mbuild
-
-.. note::
-    `PACKMOL <http://m3g.iqm.unicamp.br/packmol/>`_ is not available on pip
-    but can be installed from source or via conda.
-
 Install an editable version from source
 ---------------------------------------
 ::
 
     $ git clone https://github.com/mosdef-hub/mbuild
     $ cd mbuild
+    $ conda env create -f environment-dev.yml
+    $ conda activate mbuild-dev
     $ pip install -e .
 
 To make your life easier, we recommend that you use a pre-packaged Python
```

Comment:
This sentence might server its purpose better if we include it above the installation instructions.